1/tom-stuart-captions.srt

```diff
@@ -2619,7 +2619,7 @@ me nervous because
 565
 00:34:45,020 --> 00:34:47,140
 I've just written a
-load off actual code.
+lign of actual code.
```

Comment:
absolutely! sorry about that.

good thing you reviewed the review, I added a commit to fix that, do tell me if you would prefer a rebase.